packages/twenty-server/src/engine/core-modules/auth/auth.module.ts

```diff
@@ -114,11 +120,16 @@ import { JwtAuthStrategy } from './strategies/jwt.auth.strategy';
     RefreshTokenService,
     LoginTokenService,
     ResetPasswordService,
+    ResetMessageChannelService,
```

Comment:
auth.module should not provide business modules

Reply:
I see that this was the case before, I think we should rethink this dependency to be in the right way:
- the controller belongs to messaging (business module)
- it's calling the auth.module to handle the connection with google api (engine core module)
- then taking back the control to create the messageChannel (business module)

This way the dependency is in the other way

That's being said, this seems to be out of scope for now. I would move the services back to the business module and leave the dependecy in the wrong way for now